settings.gradle.kts

```diff
@@ -1,3 +1,5 @@
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Comment:
What's this for?

Reply:
This is to enable using project accessors like [here](https://github.com/KiltMC/Kilt/pull/112/files#diff-c0dfa6bc7a8685217f70a860145fbdf416d449eaff052fa28352c5cec1a98c06R138)